### PR TITLE
Conversion to PDFkit

### PR DESCRIPTION
**Pull request by cableman**
Ticket: https://jira.itkdev.dk/browse/BIB-644

### CODE REVIEW

File: plugins/notification/notification.js [hunk 1]
```diff
@@ -508,7 +505,7 @@ Notification.prototype.checkInReceipt = function checkInReceipt(mail, items, lan
         patronIdentifier: patronIdentifier,
         name: patronInformation.hasOwnProperty('personalName') ? patronInformation.personalName : 'Unknown',
         fines: layout.fines ? self.renderFines(mail, patronInformation.fineItems) : '',
-        loans: layout.loans ? self.renderLoans(mail, 'receipt.loans.headline', patronInformation.chargedItems, patronInformation.overdueItems) : '',
+        //loans: layout.loans ? self.renderLoans(mail, 'receipt.loans.headline', patronInformation.chargedItems, patronInformation.overdueItems) : '',
```
Comment (tuj) [inline plugins/notification/notification.js]: Remove or add a comment about why...

File: plugins/offline/offline.js [hunk 2]
```diff
@@ -380,62 +380,62 @@ Offline.prototype.checkout = function checkout(job, done) {
...
-      });
-    }, function (err) {
-      bus.emit(data.errorEvent, err);
-    });
-  });
-
-  register(null, {
-    offline: offline
-  });
+  // var bus = imports.bus;
```
Comment (tuj) [inline plugins/offline/offline.js]: Why commented out?

**Review by tuj — CHANGES_REQUESTED** [review]

File: plugins/offline/offline.js (hunk 2 above)
Comment (cableman) [reply plugins/offline/offline.js]: Fixed

File: plugins/notification/notification.js (hunk 1 above)
Comment (cableman) [reply plugins/notification/notification.js]: Fixed

**Review by tuj — APPROVED** [review]

**Pull request merged**